### PR TITLE
Update pt_Br.json added missing translations

### PR DESCRIPTION
**Pull request by evandrolacerda**
Added missing Brasilian Portuguese translations.

### CODE REVIEW

File: json/pt_BR.json [hunk 1]
```diff
@@ -166,5 +166,12 @@
     "You may delete any of your existing tokens if they are no longer needed.": "Você pode excluir seus tokens existentes se eles não forem mais necessários.",
     "You may not delete your personal team.": "Você não pode excluir o seu time pessoal.",
     "You may not leave a team that you created.": "Você não pode sair de um time que você criou.",
-    "Your email address is not verified.": "Seu endereço de e-mail não está verificado."
+    "Your email address is not verified.": "Seu endereço de e-mail não está verificado.",
+    "Administrator users can perform any action.": "Usuários com privilégios de Administrador podem executar qualquer acão.",
```
Comment (caouecs) [inline json/pt_BR.json:170]: Can you sort the tags ?

**Review by caouecs — CHANGES_REQUESTED** [review]
Obrigado

File: json/pt_BR.json (hunk 1 above)
Comment (evandrolacerda) [reply json/pt_BR.json:170]: ok, I will do

Comment (evandrolacerda) [reply json/pt_BR.json]: tags sorted, Thank you!

**Comment by caouecs** [comment]
Obrigado

**Pull request merged**